### DISCUSSION TITLE
rETH Base node stopped processing transactions after the update

### DISCUSSION THREAD

**Discussion by Denend** [paradigmxyz/reth#10673] [Help]
gm, after we updated our node to the latest rETH version - all the transaction are being stacker in a node pool, and dont go further, our 3 projects that work on that RPC are down, we need your help urgently
Thank you in advance

**Comment by onbjerg** [comment]
Can you provide more info? What transactions are stuck (tx types would be helpful, sender nonce and tx nonce as well)? Could you provide node logs?
